cms/djangoapps/models/settings/tests/test_course_metadata.py

```diff
@@ -0,0 +1,28 @@
+"""
+Tests for the CourseMetadata class
+"""
+from django.test import TestCase
+
+from adaptive_learning.config.models import AdaptiveLearningEnabledFlag
+from models.settings.course_metadata import CourseMetadata
+
+
+class TestCourseMetadata(TestCase):
+
+    def test_filtered_list_with_AdaptiveLearningEnabledFlag_set_false(self):
+        """
+        Tests to make sure that if the AdaptiveLearningEnabledFlag is set to False
+        that the adaptive_learning_configuration is filtered out of the Course Metadata list
+        """
+        AdaptiveLearningEnabledFlag.objects.create(enabled=False)
+        list = CourseMetadata.filtered_list()
+        self.assertIn('adaptive_learning_configuration', list)
+
+    def test_filtered_list_with_AdaptiveLearningEnabledFlag_set_true(self):
+        """
+        Tests to make sure that if the AdaptiveLearningEnabledFlag is set to True
+        that the adaptive_learning_configuration is not filtered out of the Course Metadata list
+        """
+        AdaptiveLearningEnabledFlag.objects.create(enabled=True)
+        list = CourseMetadata.filtered_list()
+        self.assertNotIn('adaptive_learning_configuration', list)
```

Comment:
@macornwell Thanks for adding these tests! I don't think we need to put them in a separate file, though, and a single test should be enough to cover both cases. See [`test_allow_unsupported_xblocks`](https://github.com/open-craft/edx-platform/blob/macornwell/instance-wide-disable/cms/djangoapps/contentstore/tests/test_course_settings.py#L936-L943) for an example. You can just add a `test_adaptive_learning_configuration` method below that one, following the same pattern. Then you can remove this file, as well as `cms/djangoapps/models/settings/tests/__init__.py`.